fabric/bin/peer.sh

```diff
@@ -666,7 +666,7 @@ handle_chaincode_call() {
 	if [ ! -z ${DO_WAIT_TIMEOUT+x} ]; then
 	    opt_arg="--waitForEvent ${opt_arg}"
 	fi
-	try_r $RUN ${FABRIC_BIN_DIR}/peer chaincode invoke "${OTHER_ARGS[@]}" ${opt_arg} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["__endorse", "'${encrypted_response}'"]}'
+	try_r $RUN ${FABRIC_BIN_DIR}/peer chaincode invoke "${OTHER_ARGS[@]}" ${opt_arg} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["__endorse", "'${encrypted_response}'"]}' --waitForEvent
```

Comment:
`opt_arg` (should) handle that if you passed it when invoking the cli, and if not, then we should also not do it here ...
```suggestion
	try_r $RUN ${FABRIC_BIN_DIR}/peer chaincode invoke "${OTHER_ARGS[@]}" ${opt_arg} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["__endorse", "'${encrypted_response}'"]}'
```

Reply:
done -- hopefully the auction still works

Reply:
as mentioned in the main comment.  I think the logic in `auction_test.sh` of omitting --waitForEvent is actually flawed, but that should be fixed there not in the peer cli